lib/active.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# The String class
+class String
+  # Returns true if a filename should be considered active given the current URL;
```

Comment:
Codacy found an issue: [Line is too long. [81/80]](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=6198407)